frontend/public/components/overview/project-overview.tsx

```diff
@@ -33,10 +33,10 @@ const formatCores = (cores: number): string => formatToFractionalDigits(cores, 3
 
 const overviewTooltipStyles = Object.freeze({
   content: {
-    maxWidth: '225px',
+    maxWidth: '175px',
```

Comment:
I believe the default is 170px, so we might as well not specify a custom style in this case...

But I'm not sure this is wide enough. (I realize you want to fix the nav covering the tooltip.)

Reply:
I wish react-lightweight-tooltip let you append to body :(